usama/.gitignore

```diff
@@ -0,0 +1,10 @@
+*.swp
```

Comment:
- You can use default Python .gitignore file and customize it.[gitignore](https://github.com/github/gitignore/blob/master/Python.gitignore)
- Also do add cdk.out in gitignore

Reply:
Yes, @faheemskipq I noticed that too. I am in the process of clearing my git cache and then I will pull again. Thank you so much for your review.